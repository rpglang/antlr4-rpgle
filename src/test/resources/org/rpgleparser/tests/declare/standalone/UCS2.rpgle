       DCL-S firstName UCS2(10);
       DCL-S filePath VARUCS2(5000);