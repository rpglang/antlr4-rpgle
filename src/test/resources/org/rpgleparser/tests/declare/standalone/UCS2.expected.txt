/*===TOKENS===*/
DS_Standalone            <DCL-S>
ID                       <firstName>
UCS2                     <UCS2>
'('                      <(>
NUMBER                   <10>
')'                      <)>
FREE_SEMI                <;>
DS_Standalone            <DCL-S>
ID                       <filePath>
VARUCS2                  <VARUCS2>
'('                      <(>
NUMBER                   <5000>
')'                      <)>
FREE_SEMI                <;>
/*===TREE===*/
(r
  (statement
    (dspec
      DCL-S
      (identifier (free_identifier (idOrKeyword firstName)))
      (datatype (datatypeName UCS2) (args ( (expression (number 10)) )))
      ;
    )
  )
  (statement
    (dspec
      DCL-S
      (identifier (free_identifier (idOrKeyword filePath)))
      (datatype (datatypeName VARUCS2) (args ( (expression (number 5000)) )))
      ;
    )
  )
)
/*======*/